com.github.Qv2ray.json

```diff
@@ -0,0 +1,87 @@
+{
+    "app-id": "com.github.Qv2ray",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.13",
+    "sdk": "org.kde.Sdk",
+    "command": "qv2ray",
+    "rename-desktop-file": "qv2ray.desktop",
+    "rename-icon": "qv2ray",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--socket=session-bus",
```

Comment:
Why you need access to the whole session-bus? Please run the app with `--log-session-bus` to figure out which bus access it needs.

Reply:
Hey! I want to know why the tray icon is missing on Ubuntu 18.04? I already add the necessary things at [this commit](https://github.com/ymshenyu/flathub/commit/bc0f36611f6b2de39ac071046b6910929750c711) .

Reply:
But it worked with `--log-session-bus`